Brokkr/src/main/cpp/RobotContainer.cpp

```diff
@@ -21,6 +22,8 @@ RobotContainer::RobotContainer() {
  frc::SmartDashboard::PutData("ArmPosition Mid", new ArmPosition(mArm, 0));
  frc::SmartDashboard::PutData("ArmPosition High", new ArmPosition(mArm, 0));
  frc::SmartDashboard::PutData("ArmPosition Shelf", new ArmPosition(mArm, 0));
+ frc::SmartDashboard::PutData("Wow", new SetArmPostitionForDistance(mArm, NodeLevel::HybridLevel));
```

Comment:
Really? "Wow"? 😉 

Probably should have a more descriptive label.

Reply:
Is this referring to the "Hybrid" position?